.gitignore

```diff
@@ -1,8 +1,10 @@
 DejaVuSansMono.ttf
```

Comment:
Might as well remove this old font file too, especially as it's not part of the project anymore. :)